Challenge_01/Styles/style.css

```diff
@@ -1,18 +1,22 @@
-    .signup-container {
-        padding: 25px 35px;
-        height: 100vh;
-        background-color: rgb(226, 241, 231);
-
-    }
+    :root{
+        --BackgroundColor-Page-BG : hsl(140, 35%, 92%);
```

Comment:
custom properties are better to be named in kebab case
e.g. `--background-color-page`